models/stg_snapchat_ads__campaign_geo_region_daily_report.sql

```diff
@@ -0,0 +1,63 @@
+{{ config(enabled=var('ad_reporting__snapchat_ads_enabled', true) and var('snapchat__using_campaign_region_report', false)) }}
+
+with base as (
+
+    select * 
+    from {{ ref('stg_snapchat_ads__campaign_geo_region_daily_report_tmp') }}
+),
+
+fields as (
+
+    select
+        {{
+            fivetran_utils.fill_staging_columns(
+                source_columns=adapter.get_columns_in_relation(ref('stg_snapchat_ads__campaign_geo_region_daily_report_tmp')),
+                staging_columns=get_campaign_geo_region_daily_report_columns()
+            )
+        }}
+    
+        {{ fivetran_utils.source_relation(
+            union_schema_variable='snapchat_ads_union_schemas', 
+            union_database_variable='snapchat_ads_union_databases') 
+        }}
+
+    from base
+),
+
+final as (
+
+    select
+        source_relation, 
+        campaign_id,
+        region,
```

Comment:
I noticed `regions` are lowercase 2-letter codes. Could you add an `upper()` to capitalize them?